mobile/lib/common/settings_force_close.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:get_10101/common/settings_screen.dart';
+import 'package:get_10101/common/snack_bar.dart';
+import 'package:go_router/go_router.dart';
+import 'package:get_10101/ffi.dart' as rust;
+
+class ForceClose extends StatefulWidget {
+  const ForceClose({
+    super.key,
+  });
+
+  @override
+  State<ForceClose> createState() => _ForceCloseState();
+}
+
+class _ForceCloseState extends State<ForceClose> {
+  bool isCloseChannelButtonDisabled = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+          padding: const EdgeInsets.all(20),
+          child: SafeArea(
+            child: SingleChildScrollView(
+              child: Column(
+                children: [
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Row(
+                        children: [
+                          GestureDetector(
+                            child: const Icon(
+                              Icons.arrow_back_ios_new_rounded,
+                              size: 22,
+                            ),
+                            onTap: () {
+                              context.pop();
+                            },
+                          ),
+                        ],
+                      ),
+                      const Expanded(
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            Text(
+                              "Force-Close Channel ",
+                              style: TextStyle(
+                                  fontWeight: FontWeight.w500, fontSize: 20),
+                            ),
+                          ],
+                        ),
+                      )
+                    ],
+                  ),
+                  const SizedBox(
+                    height: 20,
+                  ),
+                  Container(
+                    padding: const EdgeInsets.all(10),
+                    decoration:
+                        BoxDecoration(borderRadius: BorderRadius.circular(15)),
+                    child: RichText(
+                      text: const TextSpan(
+                        style: TextStyle(
+                            fontSize: 18,
+                            color: Colors.black,
+                            letterSpacing: 0.4),
+                        children: [
+                          TextSpan(
+                            text: "Warning",
+                            style: TextStyle(
+                                color: Colors.red, fontWeight: FontWeight.w600),
+                          ),
+                          TextSpan(
+                            text:
+                                ": Force-closing your channel should only be considered as a last resort if 10101 is not reachable.\n\n\n",
```

Comment:
Two new lines are enough :) 

```suggestion
                                ": Force-closing your channel should only be considered as a last resort if 10101 is not reachable.\n\n",
```